docs/spec/src/protocol/payments/payment_system.md

```diff
@@ -0,0 +1,178 @@
+# EigenDA Payment System
+
+## 1. Overview
+
+The EigenDA payment system allows users to pay for blob dispersals through two methods: reservations and on-demand
+payments. All payment logic is implemented in the [`core/payments`](../../../../../core/payments/) package.
+
+**Key Concepts:**
+- Blob sizes are measured in *symbols*, where each symbol contains 32 bytes of data
+- Blob sizes are measured **post-blob encoding**: user payloads expand to some degree during blob encoding
+- Blob sizes are constrained to powers-of-two: dispersals are rounded up to the next power-of-two number of symbols
+  when computing size
+- The [PaymentVault](https://github.com/Layr-Labs/eigenda/blob/master/contracts/src/core/PaymentVault.sol) contract
+  stores all on-chain payment-related data:
+  - User reservation parameters
+  - User on-demand deposits
+  - Global payment parameters, including but not limited to:
+    - `minNumSymbols`: dispersals smaller than this threshold are billed as if they were `minNumSymbols` in size
+    - `pricePerSymbol`: the price per symbol (in wei) for on-demand payments
+
+## 2. Payment Methods
+
+### 2.1 Reservation Payments
+
+- Reservations provide guaranteed bandwidth for a specified time period.
+   - Users reserve capacity in advance, and must "use it or lose it".
+   - Reservations are procured out-of-band, through EigenDA
+- The system uses a [leaky bucket algorithm](../../../../../core/payments/reservation/leaky_bucket.go) to manage usage:
+  symbols are added to the bucket each time a blob is dispersed, and these leak out over time. A user can only make a
+  dispersal if the leaky bucket has available capacity.
+   - The total capacity of the leaky bucket is defined in terms of a *duration*. The size of the bucket in symbols is
```

Comment:
total capacity is not in duration, its in symbols/s.
It is parametrized by BOTH duration AND rate.

Reply:
It is defined in code by passing in a duration, but I see how this could be confusing. Reworded.

005a8438a6d29128d9bb1027c5ad47a71cc18493